windows/installer-source/KaliteSetupScript.iss

```diff
@@ -242,14 +242,42 @@ begin
     end;
 end;
 
-procedure HandleUpgrade(targetPath : String);
+procedure SelectContentFolder;
 var
-    prevVerStr : String;
     userKaliteContent: String;
     systemKaliteDir: String;
     userPath : String;
     contentPath : String;
-    contentDb : String;
+    retCode: Integer;
+begin
+    userPath := ExpandConstant('{%USERPROFILE}')
+    contentPath := ExpandConstant('{%CONTENT_ROOT}')
+    systemKaliteDir := ExpandConstant(userPath + '\.kalite')
+    userKaliteContent := ExpandConstant(systemKaliteDir + '\content');
+    if Not DirExists(userKaliteContent) then
+    begin
+     if Not DirExists(contentPath) then
+        begin

```

Comment:
I this condition you are saying: 

> If there is no user content folder AND there is no previously declared `CONTENT_ROOT` environment

But this only matches that there already

1. was a previous installation AND
1. the previous installation does not have a content folder in the default location
1. whatever content folder was selected previously doesn't exist

I think for a user to go through the installation and then being asked questions, and then if you try to re-install and *not* get asked the same questions might be problematic? If they want to correct a mistake for instance, they have to go through the whole uninstallation (in which I'm not sure if removal of environment registry is guaranteed?)

My suggestion would be to completely remove the condition and instead display the path to the user.

In the future, we might also display a count of free space available on the drive.